docs/data-management/intro.md

```diff
@@ -0,0 +1,122 @@
+After data collection, a significant amount of post-scanner steps remain to reach the point where all data and derivatives have been processed, making sure they are BIDS compliant and ready for release.
+
+The flowchart below shows an overview of the post-scanner data workflow:
+
+``` mermaid
+flowchart TB
+    classDef imp stroke:#f00,stroke-width:2px;
+
+    subgraph PACS[PACS]
+    end
+
+    subgraph physio["Physio-recording Laptop ({{ secrets.hosts.acqknowledge | default("███") }})"]
+    end
+
+    subgraph et_pc[Eye-tracker Computer]
+    end
+
+    subgraph psychopy_pc["Stimuli presentation Laptop ({{ secrets.hosts.psychopy | default("███") }})"]
+    end
+
+    subgraph local_data[Local Storage]
+        raw_mri[Raw MRI]
+        bids_mri[BIDS MRI]
+        bids_phys[BIDS physio]
+        bids_edf[BIDS edf]
+    end
+
+    subgraph dropbox[Dropbox]
+        raw_phys[Raw physio]
+        raw_edf[Raw edf]
+    end
+    
+    subgraph datalad[DataLad]
+        bids_mri_datalad[BIDS MRI]
+        bids_phys_datalad[BIDS physio]
+        bids_edf_datalad[BIDS edf]
+        %%bids_phys_datalad[BIDS physio]
+
+        mriqc_out[Visual reports]
+        preproc_fMRI[Preprocessed fMRI]
+        preproc_dMRI[Preprocessed dMRI]
+
+        FC
+
+        dti_out[dMRI derivatives]
+        SC
+    end
+    
+    subgraph openneuro[Open Neuro Datasets]
+        unprocessed[Unprocessed Dataset]
+        processed[Preprocessed Dataset]
+        connectivity[SC, FC]
+
+        rel_1[HCPh Release 1]
+        rel_2[HCPh Release 2]
+        rel_3[HCPh Release 3]
+    end
+
+    PACS --->|pacsman Script| raw_mri
+    physio -->|Dropbox Sync| raw_phys
+    et_pc -->|Psychopy| psychopy_pc
+    psychopy_pc --> |Dropbox Sync| raw_edf
+    
+    raw_mri -->|HeudiConv script| bids_mri
+    raw_phys --->|phys2bids script| bids_phys
+    raw_edf ---> |edf2bids script| bids_edf
+
+    bids_mri -->|FIRST session only !| med[Clinical screening]
+    med -->|"Incidental(s)"| alert[Alert participant]
+    class alert imp
+
+    local_data --->|cron Job| HOrUs
+    HOrUs ---> datalad
+    bids_mri_datalad -->|MRIQC| mriqc_out[Visual Reports]
+
+    mriqc_out -->|Q'Kay| mri_qc_screen[Screening]
+    mri_qc_screen -->|QCs not ok| Reschedule
+    class Reschedule imp
+
+    bids_mri_datalad -->|fMRIPrep| preproc_fMRI[Preprocessed fMRI]
+    bids_mri_datalad -->|dMRIPrep| preproc_dMRI[Preprocessed dMRI]
+    
+    bids_mri_datalad -->|pydeface Script| unprocessed
```

Comment:
Neat pick comment but can you make the capital letters consistent? always capital on first words and either always or never capital on subsequent words. Thanks :)

Reply:
I wanted to keep the scripts name as they are but I'll double check

Reply:
Added italic for script names and fixed inconsistencies